### PR TITLE
Add op-sepolia to manifest file names

### PR DESCRIPTION
**Pull request by cruzdanilo**

### CODE REVIEW

File: packages/core/src/provider.ts [hunk 1]
```diff
@@ -143,6 +143,7 @@ export const networkNames: { [chainId in number]?: string } = Object.freeze({
   43114: 'avalanche',
   42220: 'celo',
   44787: 'celo-alfajores',
+  11155420: 'op-sepolia',
   11155111: 'sepolia',
```
Comment (Amxx) [inline packages/core/src/provider.ts:147]: IMO they should be sorted by chainId
```suggestion
  11155111: 'sepolia',
  11155420: 'op-sepolia',
```

**Review by ericglau — COMMENTED** [review]
Thanks for the PR.

Can you please a [changelog entry](https://github.com/OpenZeppelin/openzeppelin-upgrades/blob/master/packages/core/CHANGELOG.md) at the top with heading `## Unreleased`?

See https://github.com/OpenZeppelin/openzeppelin-upgrades/blob/master/packages/core/CHANGELOG.md#1250-2023-04-26 for example.

**Review by Amxx — COMMENTED** [review]
420 is marked as "optimism-goerli"

Should this be "optimism-sepolia" ? Is there a strong convension to use "op-" vs "optimism-" ?

**Comment by cruzdanilo** [comment]
i understand there's no way to easily rename current networks (the "Optimism" network is now called "OP Mainnet"), but i feel like new networks should use the proper names.

**Review by ericglau — APPROVED** [review]

**Pull request merged**